document custom logo (#197)
carpentries/sandpaper-docs: episodes/editing.md

carpentry
: The code for the specific carpentry that the lesson belongs to (swc, dc, lc, cp)

::::::::::: callout

### Adding a custom logo

> The `carpentry` variable works with the `varnish` package to control the logo displayed on your lesson. You can display your organisation's logo by

Reviewer: ```suggestion
The "carpentry" variable works with the {varnish} package to control the logo displayed on your lesson. You can display your own logo by
```

I felt that 'organisation' was too specific, e.g. it could be a project logo instead.

> 1. Adding your organisation's logo file (e.g. 'ice-cream-logo.svg') to your fork of `varnish` in the `inst/pkgdown/assets/assets/images` folder. 

Reviewer: ```suggestion
1. Adding the logo file as SVG (e.g. 'ice-cream-logo.svg') to your fork of {varnish} in the `inst/pkgdown/assets/assets/images` folder. 
```

> 1. Setting `carpentry` to match the name of your logo file. (e.g. 'ice-cream')

Reviewer: ```suggestion
1. Setting "carpentry" to match the beginning of the name of your logo file. E.g. to use the `ice-cream-logo.svg` file given above, "carpentry" should be set to 'ice-cream'.
```

carpentry
: The code for the specific carpentry that the lesson belongs to (swc, dc, lc, cp)

> carpentry_description
> : (Optional) Full organisation name. Not needed when carpentry is swc, dc, lc, or cp. 

Reviewer: ```suggestion
carpentry_description
: (Optional) Full organisation name. Not needed when carpentry is swc, dc, lc, cp, incubator, or lab. 
```
